doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -381,22 +381,29 @@ Note that this behavior is off by default.
 Advanced configuration
 ----------------------
 
-Setting the intermediate local directory
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Setting the local staging directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When a ``storage_path`` is specified, training outputs are saved to an
-*intermediate local directory*, then persisted (copied/uploaded) to the ``storage_path``.
-By default, this intermediate local directory is a sub-directory of ``~/ray_results``.
+Apart from files written to the ``storage_path``, Ray Train also writes to some
+logfiles and metadata files in a *local staging directory* before they get
+persisted (copied/uploaded) to the ``storage_path``.
+By default, this local staging directory is a sub-directory of the temporary Ray session
+directory (``/tmp/ray/session_latest``), which is also where Ray Core logs are dumped.
```

Comment:
Not specific to logs since that would be in the `/logs` sub-directory.
```suggestion
directory (``/tmp/ray/session_latest``), which is also where other temporary Ray files are dumped.
```